.github/workflows/pr-test.yml

```
@@ -132,7 +112,7 @@ jobs:
           # directory.
           # The purpose of this is for the PR Review Companion to later
           # be able to use this raw diff file for the benefit of analyzing.
-          git diff --diff-filter=ACMRT ${{ github.event.pull_request.base.sha }} ${{ github.sha }} > $BUILD_OUT_ROOT/DIFF
+          wget "https://github.com/${{ github.repository }}/pull/${{ github.event.number }}.diff" -O $BUILD_OUT_ROOT/DIFF
```

download diff file from github

---

Could we use GitHub CLI for this?

```suggestion
          gh pr diff > $BUILD_OUT_ROOT/DIFF
```